Leaf API.postman_collection.json

```diff
@@ -1930,7 +1930,7 @@
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n    \"operationsImageCreation\": false,\n    \"operationsSummaryCreation\": false\n}"
+											"raw": "{\n    \"operationsImageCreation\": false,\n    \"geoimagesResolution\": 0.00001,\n    \"geoimagesShape\": \"SQUARE\",\n    \"geoimagesProjection\": \"EPSG:3857\",\n    \"geoimagesColorRamp\": {\n        \"0%\"  : [200,   0, 0],\n        \"35%\" : [255,  40, 0],\n        \"45%\" : [255, 150, 0],\n        \"55%\" : [255, 240, 0],\n        \"65%\" : [  0, 230, 0],\n        \"75%\" : [  0, 190, 0],\n        \"100%\": [  0, 130, 0],\n        \"nv\"  : [  0,   0, 0, 0]\n    },\n    \"fieldsAutoSync\": true,\n    \"fieldsAttachIntersection\": 10,\n    \"fieldsMergeIntersection\": 10\n}"
```

Comment:
Can we keep the value of the properties to the default value of 0.01?

Reply:
Sure. Done in 336642efbc36d289772c3e8f712f4eedcfe53440